templates/adherent/dashboard.html.twig

```diff
@@ -0,0 +1,213 @@
+                    {% if app.user.interests|length > 0 %}
+                        <ul>
+                            {% for interest in app.user.interests %}
+                                <li class="dashboard-tag">{{ interest|capitalize }}</li>
+                            {% endfor %}
+                        </ul>
+                        <br>
+                        <p>
+                            <a href="{{ interestEditPath }}" class="link--no-decor link--newblue">{{ 'dashboard.edit'|trans }}</a>
+                        </p>
+                    {% else %}
+                        {{ 'dashboard.interests.empty'|trans({'%url%': interestEditPath})|raw }}
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+
+        <div class="dashboard__column-right">
+            <div class="shortcuts">
+                <b>{{ 'dashboard.shortcut.label'|trans }}</b>
+                <hr>
+                <ul>
+                    {% if is_granted('ROLE_PROCURATION_MANAGER') %}
+                        <a href="{{ path('app_procuration_manager_requests') }}" class="link--no-decor link--newblue">{{ 'dashboard.shortcut.procuration_manager_space'|trans }}</a>
+                    {% endif %}
+                    {% if is_granted('ROLE_COORDINATOR_COMMITTEE') %}
+                        <a href="{{ path('app_coordinator_committees') }}" class="link--no-decor link--newblue">{{ 'dashboard.shortcut.committee_coordinator_space'|trans }}</a>
+                    {% endif %}
+                    {% if is_granted('ROLE_COORDINATOR_CITIZEN_PROJECT') %}
+                        <a href="{{ path('app_coordinator_citizen_project') }}" class="link--no-decor link--newblue">{{ 'dashboard.shortcut.citizen_project_coordinator_space_space'|trans }}</a>
+                    {% endif %}
+                    {% if app.user.isCitizenProjectAdministrator %}
+                        <li><a href="{{ path('app_citizen_project_image_generator') }}" class="link--no-decor link--newblue">{{ 'dashboard.shortcut.leaflets_posters'|trans }}</a></li>
+                    {% endif %}
+                    <li><a href="{{ path('app_documents_index') }}" class="link--no-decor link--newblue">{{ 'dashboard.shortcut.my_documents'|trans }}</a></li>
+
+                    {% if is_granted('ROLE_REFERENT') %}
+                        <li><a href="https://en-marche.helprace.com/" target="_blank" class="link--no-decor link--newblue">{{ 'dashboard.shortcut.contact'|trans }}</a></li>
+                    {% endif %}
+                </ul>
+            </div>
+            <div class="skills">
+                {% set skillEditPath = path('app_summary_manager_step', ({step: 'skills'})) %}
+                <b>{{ 'dashboard.my_skills'|trans }}</b>
+                <hr>
+                {% if summary is not null and summary.skills|length > 0 %}
+                    <ul>
+                        {% for skill in summary.skills|slice(0,5) %}
```

Comment:
CS: `0, 5` and for other `slice` calls :)

Reply:
done